libprjoxide/prjoxide/src/bels.rs

```diff
@@ -584,6 +584,25 @@ impl Bel {
         }
     }
 
+    pub fn make_dcs() -> Bel {
+        // FIXME: Support DCS1 for CertusPro-NX too
+        let postfix = format!("DCS_DCSIP");
+        Bel {
+            name: format!("DCS0"),
+            beltype: format!("DCS"),
+            pins: vec![
+                input!(&postfix, "CLK0", "Clock input port 0 (default)"),
+                input!(&postfix, "CLK1", "Clock input port 1"),
+                input!(&postfix, "SEL", "Input clock select"),
+                input!(&postfix, "SELFORCE", "Selects glitchless (0) or non-glitchless (1) behavior"),
+                output!(&postfix, "DCSOUT", "Clock output port"),
+            ],
+            rel_x: 0,
+            rel_y: 0,
+            z: 0
```

Comment:
this should probably be `z=4` so it doesn't overlap with the DCCs at the same tile?

Reply:
Ah, right. I missed that, thanks!